src/main.c

```diff
@@ -3221,7 +3221,8 @@ static int openDatabase(
   sqlite3 **ppDb,        /* OUT: Returned database handle */
   unsigned int flags,    /* Operational flags */
   const char *zVfs,      /* Name of the VFS to use */
-  const char *zWal       /* Name of WAL module to use */
+  const char *zWal,      /* Name of WAL module to use */
+  void* pMethodsData            /* Pointer to the user data*/
```

Comment:
let's be explicit with naming here and call it pWalMethodsData - this function is not WAL-specific